load_tests/daemonset/cloudwatch.yaml

```diff
@@ -0,0 +1,248 @@
+        Match               kube.*
+        region              us-west-2
+        log_group_name      $CW_LOG_GROUP_NAME
+        log_stream_prefix   load-test-fluent-bit-
+        auto_create_group   false
+        log_key             log
+        workers             1
+        auto_retry_requests true
+        net.keepalive       off
+
+  parsers.conf: |
+    [PARSER]
+        Name                docker
+        Format              json
+        Time_Key            time
+        Time_Format         %Y-%m-%dT%H:%M:%S.%LZ
+
+    [PARSER]
+        Name                syslog
+        Format              regex
+        Regex               ^(?<time>[^ ]* {1,2}[^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?(?:[^\:]*\:)? *(?<message>.*)$
+        Time_Key            time
+        Time_Format         %b %d %H:%M:%S
+
+    [PARSER]
+        Name                container_firstline
+        Format              regex
+        Regex               (?<log>(?<="log":")\S(?!\.).*?)(?<!\\)".*(?<stream>(?<="stream":").*?)".*(?<time>\d{4}-\d{1,2}-\d{1,2}T\d{2}:\d{2}:\d{2}\.\w*).*(?=})
+        Time_Key            time
+        Time_Format         %Y-%m-%dT%H:%M:%S.%LZ
+
+    [PARSER]
+        Name                cwagent_firstline
+        Format              regex
+        Regex               (?<log>(?<="log":")\d{4}[\/-]\d{1,2}[\/-]\d{1,2}[ T]\d{2}:\d{2}:\d{2}(?!\.).*?)(?<!\\)".*(?<stream>(?<="stream":").*?)".*(?<time>\d{4}-\d{1,2}-\d{1,2}T\d{2}:\d{2}:\d{2}\.\w*).*(?=})
+        Time_Key            time
+        Time_Format         %Y-%m-%dT%H:%M:%S.%LZ
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ds-cloudwatch-$THROUGHPUT
+  namespace: load-test-fluent-bit-eks-ns
+  labels:
+    k8s-app: load-test-fluent-bit-eks
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  selector:
+    matchLabels:
+      k8s-app: load-test-fluent-bit-eks
+  template:
+    metadata:
+      labels:
+        k8s-app: load-test-fluent-bit-eks
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - name: fluent-bit
+        image: $FLUENT_BIT_IMAGE
+        imagePullPolicy: Always
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh",  "-c", "sleep 10"]
+        resources:
+          limits:
+            memory: 1Gi
+          requests:
+            cpu: 1.0
+            memory: 1Gi
+        volumeMounts:
+        # Please don't change below read-only permissions
+        - name: fluentbitstate
+          mountPath: /var/fluent-bit/state
+        - name: fluentbitcloudwatch$THROUGHPUT
+          mountPath: /var/fluent-bit/cloudwatch/$THROUGHPUT
+        - name: varlog
+          mountPath: /var/log
+          readOnly: true
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+        - name: cloudwatch-$THROUGHPUT-config
+          mountPath: /fluent-bit/etc/
```

Comment:
why is $THROUGHPUT in these names and paths?

Reply:
It's for this line: https://github.com/aws/aws-for-fluent-bit/blob/d4c55cac1794d04803e40ecea9281af5492a6508/load_tests/daemonset/cloudwatch.yaml#L34 and https://github.com/aws/aws-for-fluent-bit/blob/d4c55cac1794d04803e40ecea9281af5492a6508/load_tests/daemonset/cloudwatch.yaml#L239.  Different throughput tests need different configs so I make it different name (we need different configmap for different tests, otherwise exiting tests will be modified)